src/workloads/execution/AggregateExpressions.yml

```diff
@@ -0,0 +1,1224 @@
+          OperationCommand:
+            aggregate: *Collection
+            pipeline: [{$project: {acos: { $acos: $doubleBounded}}}]
+            cursor: {}
+
+- Name: ExpressionArcSine
+  Type: RunCommand
+  Database: *Database
+  Threads: *Threads
+  Phases:
+    OnlyActiveInPhases:
+      Active: [10]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: *Repeat
+        Database: *Database
+        Collection: *Collection
+        Operations:
+        - OperationMetricsName: AggExressionSine
+          OperationName: RunCommand
+          OperationCommand:
+            aggregate: *Collection
+            pipeline: [{$project: {asin: { $asin: $doubleBounded}}}]
+            cursor: {}
+
+- Name: ExpressionArcTangent
+  Type: RunCommand
+  Database: *Database
+  Threads: *Threads
+  Phases:
+    OnlyActiveInPhases:
+      Active: [10]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: *Repeat
+        Database: *Database
+        Collection: *Collection
+        Operations:
+        - OperationMetricsName: AggExressionArcTangent
+          OperationName: RunCommand
+          OperationCommand:
+            aggregate: *Collection
+            pipeline: [{$project: {atan: { $atan: $double}}}]
+            cursor: {}
+
+- Name: ExpressionArcTangent2
+  Type: RunCommand
+  Database: *Database
+  Threads: *Threads
+  Phases:
+    OnlyActiveInPhases:
+      Active: [10]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: *Repeat
+        Database: *Database
+        Collection: *Collection
+        Operations:
+        - OperationMetricsName: AggExressionArcTangent2
+          OperationName: RunCommand
+          OperationCommand:
+            aggregate: *Collection
+            pipeline: [{$project: {atan2: { $atan2: [$double, $doubleBounded]}}}]
+            cursor: {}
+
+- Name: ExpressionHyperbolicArcTangent
+  Type: RunCommand
+  Database: *Database
+  Threads: *Threads
+  Phases:
+    OnlyActiveInPhases:
+      Active: [10]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: *Repeat
+        Database: *Database
+        Collection: *Collection
+        Operations:
+        - OperationMetricsName: AggExpressionHyperbolicArcTangent
+          OperationName: RunCommand
+          OperationCommand:
+            aggregate: *Collection
+            pipeline: [{$project: {atanh: { $atanh: $double}}}]
+            cursor: {}
+
+- Name: ExpressionHyperbolicArcCosine
+  Type: RunCommand
+  Database: *Database
+  Threads: *Threads
+  Phases:
+    OnlyActiveInPhases:
+      Active: [10]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: *Repeat
+        Database: *Database
+        Collection: *Collection
+        Operations:
+        - OperationMetricsName: AggExpressionHyperbolicArcCosine
+          OperationName: RunCommand
+          OperationCommand:
+            aggregate: *Collection
+            pipeline: [{$project: {acosh: { $acosh: $double}}}]
```

Comment:
needs to be between 1 and infinity: https://www.mongodb.com/docs/manual/reference/operator/aggregation/acosh/

Reply:
Done.